pkg/api/v20191231preview/register.go

```diff
@@ -31,5 +31,16 @@ func init() {
 		OpenShiftClusterCredentialsConverter: func() api.OpenShiftClusterCredentialsConverter {
 			return &openShiftClusterCredentialsConverter{}
 		},
+		OperationList: api.OperationList{
```

Comment:
Do we have to upstream these additions to `azure-rest-api-specs`?

Reply:
No.  I believe they're default operations that can be performed on any resource provider with long-running operations hence nothing to update.